react/react_intro/README.md

```diff
@@ -44,7 +44,7 @@ There are at least four entirely distant components that are on the page includi
 - the stories feed
 - the contacts list.  
 
-Using our DOM manipulation approach, we would have all of these different elements in a single html file.  This would be challenging to maintain because we a bug in one area could easily affect the rest of the page.  
+Using our DOM manipulation approach, we would have all of these different elements in a single html file.  This would be challenging to maintain because a bug in one area could easily affect the rest of the page.
 
 React solves this problem by introducing `Components`.  Instead of laying out all of our HTML in a single file, we can separate it into separate classes and combine those classes together.
```

Comment:
I'm not crazy about saying that the main benefit of React is that we can use separate files:
1. I don't think this was the main reason why Facebook created React and, to me, it isn't the main reason why React is better or more popular than vanilla JS or other frameworks.
2. Many other previous libraries and frameworks allowed for separate files, and you can even do it with modern vanilla JS
3. In a way this isn't quite true since webpack ends up stuffing everything into a single JS file (unless a user carefully configures it not to do that)

I see the key plusses of React as
- React makes DOM manipulation more performant thanks to the virtual DOM
- React takes away a lot of the headaches of DOM manipulation with vanilla JS (more expressive, fewer lines of code, fewer event-driven headaches, don't have to find a specific DOM node to attach each element to)
- React makes it easier to combine and reuse interactive components

Is anyone up for taking on a larger rewrite of this section?

Reply:
Yes I had a some the same feelings about that line in particular. I can make these changes for sure, I like that what you have listed here as benefits to using React